.github/workflows/pr-checks.yaml

```
@@ -12,12 +12,16 @@ on:
 jobs:
   build-test-lint:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x, 20.x]
```

[nitpick] I always like the normal list notation better in YAML:
```suggestion
        node-version:
          - 16.x
          - 18.x
          - 20.x
```

---

What even is yaml?